reference/features/search_parameters.md

```diff
@@ -42,6 +42,12 @@ When `q` isn't specified, a **placeholder search** is performed. Placeholder sea
 
 Placeholder search returns the documents that best match other search parameters, sorted according to that index's [ranking rules](/reference/features/settings.md#ranking-rules). This feature also supports [faceting](/reference/features/faceted_search.md) and [filtering](/reference/features/filtering.md)
 
+### Phrase search
+
+You can enclose search terms in double quotes (`"`), in which case MeiliSearch will only return documents containing all terms in the order they were given. This is called a [phrase search](/learn/what_is_meilisearch/features.md#phrase-search).
+
+Phrase searches are case-insensitive and ignore soft separators such as `-`, `,`, and `:`. You can mix them with regular queries: MeiliSearch will first fetch all documents with exact matches and [then proceed with its default behavior](/learn/core_concepts/relevancy.md).
```

Comment:
```suggestion
Phrase searches ignore soft separators such as `-`, `,`, and `:`. You can mix them with regular queries: MeiliSearch will first fetch all documents with exact matches and [then proceed with its default behavior](/learn/core_concepts/relevancy.md).
```

Reply:
Thanks for the input, @curquiza! Does this mean we are deviating from the spec? 

For reference: [phrase query spec, usage heading](https://github.com/meilisearch/specifications/blob/main/text/0043-phrase-query.md#phrase-query-usage)

Reply:
Indeed @ManyTheFish can you confirm the mistake?
FYI @gmourier

Reply:
Phrase searches are case-insensitive.

Reply:
Sorry! I confused case-sensitive and case-insensitive!!!! My bad everyone! 😂 I definitely need holidays!

Reply:
Should we link to the full list of soft separators?

Reply:
I thought about that, but couldn't find an in-house resource. Do you have a good external resource we could link to?

Reply:
We have a documentation page in the "Under the hood" section that describes our soft and hard separators https://docs.meilisearch.com/reference/under_the_hood/datatypes.html but it does not describe the full list.

Here the source code https://github.com/meilisearch/tokenizer/blob/0db4b8fca263bfc89cb4c4453877a95e63806a73/src/detection.rs#L49

@ManyTheFish Please tell me if I'm wrong about the file 😇. Also, I didn't see the `—` char we identified some time ago. We had added an issue but it seems to me that there was a mix-up do you recall?

Reply:
@gmourier thanks for the link! 

Is there a reason why the datatypes page doesn't list all separators? Do we use different lists of soft/hard separators depending on what we're doing?

If not, I'll open an issue and update it.